pkgs/by-name/co/cook-cli/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildNpmPackage,
+  rustPlatform,
+  pkg-config,
+  openssl,
+  darwin,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "cook-cli";
+  version = "0.7.0";
+  src = source;
```

Comment:
This is unnecessary. Just do `src = fetchFromGitHub { ... }` directly.

Reply:
Ah, of course. Thanks.

I've made all the requested changes, could you take another look?